code/game/objects/items/quickdeploy_cade.dm

```diff
@@ -0,0 +1,37 @@
+//An item that allows a very quick deployment of a barricade
+/obj/item/quikdeploy_cade
```

Comment:
```suggestion
/obj/item/quikdeploy/cade
```

You could make this a whole set of `QuikDeploy:tm:` items.

Optional, and up to you though

Reply:
Good suggestion.